# Fields
float32 alpha
float32 beta
float32 sizeX
float32 sizeY
float32 heading
uint8 id
